gen-screenshots/swayconfig_i3rs

```diff
@@ -4,6 +4,7 @@ bar {
     status_padding 0
     tray_output none
     font pango:DejaVu Sans Mono, Font Awesome 6 Free 10
+    #font pango:DejaVu Sans Mono for Powerline, Font Awesome 6 Free 10
```

Comment:
I'm not sure why, but I had an extra space between the separator and the text, unless I used the Powerline version of DejaVu.